packages/api/cms-api/README.md

```diff
@@ -0,0 +1,26 @@
+# CMS-API
+
+## Sentry Module
+
+```ts
+// main.ts
+
+app.use(Sentry.Handlers.requestHandler());
+app.use(Sentry.Handlers.tracingHandler());
+app.use(Sentry.Handlers.errorHandler());
+```
+
+```ts
+// app.module.ts
+
+SentryModule.forRoot({
+    dsn: "sentry_dsn_url",
+    enviroment: "dev",
+}),
+```
+
+Please use @sentry/wizard to configure uploading source maps
+
+```
+npx @sentry/wizard@latest -i sourcemaps
+```
```

Comment:
was this meant for the docs?

Reply:
Yes, please move to the docs

Reply:
Done https://github.com/vivid-planet/comet/pull/2334/commits/ce291b19e4d8872a96167f76e49f7faaed6fe4e4